forte/core.py

```diff
@@ -66,3 +69,40 @@ def __new__(cls):
 
     def __del__(cls):
         forte.cleanup()
+
+
+def start_logging():
+    """This function starts logging"""
```

Comment:
The docstring can be more specific: sets log output to a file, and sets the log level to all information. (Default is warnings, errors, and critical errors, which we signal by terminating Forte.) `flog` below may make this distinction important.

Reply:
Another good point. A more detailed explanation added.